jmh-core/src/main/java/org/openjdk/jmh/util/Utils.java

```diff
@@ -590,7 +590,22 @@ public static Properties readPropertiesFromCommand(List<String> cmd) {
 
             errDrainer.join();
             outDrainer.join();
-            out.loadFromXML(new ByteArrayInputStream(baos.toByteArray()));
+            try {
+                out.loadFromXML(new ByteArrayInputStream(baos.toByteArray()));
+            } catch (InvalidPropertiesFormatException ex) {
+                // Maybe some VM output has preceded the XML content?
+                String output = new String(baos.toByteArray(), Charset.defaultCharset());
```

Comment:
I suspect that since we are reading from the console, we need to use `Utils.guessConsoleCharset()` instead of trusting the default one.